### PR TITLE
Solve Validation Errors for Quizzify

### PR DESCRIPTION
**Pull request by AaronSosaRamos**
This PR is made for solving Issue #48.
What I've discovered in my research of this problem was that a malformed JSON was made when the number of chunks was generally bigger. For example, I've found this for the actual details that @mikhailocampo provided:
image
image
I've also tested them with the following files and found the same problem:
https://hagan.okstate.edu/NNDesign.pdf with the topic "Neural Networks"
image
https://diposit.ub.edu/dspace/bitstream/2445/180441/2/tfm_lichtner_bajjaoui_aisha.pdf
image
So, I've discovered that the malformed JSON had a "properties" key in its composition, making it unappropriate for the actual Pydantic Schema. So, I developed a function to test if the response has that. If so, then it will re-try to generate another question, providing appropriate results with bigger PDF files.
image
image
image

### CODE REVIEW

**Comment by mikhailocampo** [comment]
Although accurate, I think this is separate from the #48 Issue given that the issue encountered here is related to `ValidationErrors` rather than the mapping error from #48

File: app/features/quizzify/tools.py
Comment (mikhailocampo) [inline app/features/quizzify/tools.py:1]: What is the primary difference between `validate_question` and `has_format` ? I think the multiple functions do follow single responsibility but perhaps the complexity and readability for this feature has gone down

Comment (AaronSosaRamos) [reply app/features/quizzify/tools.py:1]: I'm agree with you @mikhailocampo. I've removed the `has_format` method as it's not necessary anymore.

Comment (AaronSosaRamos) [reply app/features/quizzify/tools.py:1]: What I done is to only add a condition for reviewing if the response collapses and contains the `properties` attribute. If it has that attribute, the system re-tries to do another validate question. It is working fine.

Comment (AaronSosaRamos) [reply app/features/quizzify/tools.py:1]: It's currenly working with the provided scenario:
image

**Comment by AaronSosaRamos** [comment]
Closed due we solved that issue in the new `Develop` branch

**Pull request closed without merging**